addons/account/tests/test_account_move_entry.py

```diff
@@ -485,3 +486,138 @@ def test_misc_prevent_unlink_posted_items(self):
         # You can remove journal items if the related journal entry is draft.
         self.test_move.button_draft()
         self.test_move.line_ids.unlink()
+
+    def test_invoice_like_entry_reverse(self):
+        """ Reverse entries of entries with invoice like structure (balance, tax), must have its
+        tax tags set accordingly to repartition lines same as invoices.
+        """
+        def get_tags(name):
+            tags = defaultdict(dict)
+            for line_type, repartition_type in [(l, r) for l in ('invoice', 'refund') for r in ('base', 'tax')]:
+                tags[line_type][repartition_type] = self.env['account.account.tag'].create({
+                    'name': '%s %s %s' % (name, line_type, repartition_type),
+                    'applicability': 'taxes',
+                    'country_id': self.env.ref('base.us').id,
+                })
+            return tags
+
+        def get_tax(name, tags, invoice_tax_account, refund_tax_account):
+            return self.env['account.tax'].create({
+                'name': name,
+                'amount': 10.0,
+                'invoice_repartition_line_ids': [
+                    (0, 0, {
+                        'factor_percent': 100,
+                        'repartition_type': 'base',
+                        'tag_ids': [(6, 0, tags['invoice']['base'].ids)],
+                    }),
```

Comment:
Just a suggestion, but why not use the new syntax introduced here https://github.com/odoo/odoo/commit/eded14b4c439714bf172cc479e5ecd3f3ddbaf2b ?
```suggestion
                'invoice_repartition_line_ids': [
                    Command.create({
                        'factor_percent': 100,
                        'repartition_type': 'base',
                        'tag_ids': [(6, 0, tags['invoice']['base'].ids)],
                    }),
```

Reply:
Because we're in 13.0 here ;)

Reply:
Makes sense :sweat_smile: